llpc/context/llpcPipelineContext.cpp

```diff
@@ -342,6 +342,10 @@ Options PipelineContext::computePipelineOptions() const {
   options.enablePrimGeneratedQuery = getPipelineOptions()->enablePrimGeneratedQuery;
   options.enableFragColor = getPipelineOptions()->enableFragColor;
 
+  options.rtBoxSortHeuristicMode = m_rtState.boxSortHeuristicMode;
+  options.rtStaticPipelineFlags = m_rtState.staticPipelineFlags;
+  options.rtTriCompressMode = m_rtState.triCompressMode;
```

Comment:
Do you need to default-initialize useGpurt here as well?

Reply:
The `= {}` initialization at the top should take care of that, right?

Reply:
Yes. My question is based on the fact that we initialize other members with a constant value here as well, so it is merely a question about coding style.

Reply:
I think we don't need to because it is always false (unknown) at here.